src/field.jl

```diff
@@ -741,3 +741,22 @@ function tryparsenext(f::Field{T}, str, i, len, opts) where {T}
     return R(convert(T, res)), i
 end
 
+struct BooleanToken{T} <: AbstractToken{T}
+end
+
+function BooleanToken(t::Type{T}) where T
+    BooleanToken{T}()
+end
+
+fromtype(::Type{Bool}) = BooleanToken(Bool)
+
+function tryparsenext(::BooleanToken{Bool}, str, i, len, opts)
+    value, ii = tryparsenext(StringToken(String), str, i, len, opts)
+    if value.value in ("true", "True")
```

Comment:
Should we be using the same `default_true_strings ` and `default_false_strings ` as in `guesstype.jl`?